lib/client.js

```diff
@@ -11,6 +11,8 @@ const QoSPacket = require('./qos-packet')
 const handleSubscribe = require('./handlers/subscribe')
 const handleUnsubscribe = require('./handlers/unsubscribe')
 const handle = require('./handlers')
+const through = require('through2')
```

Comment:
You can just use a `new Transform({ transform() {} }})` instead.

Reply:
What are the benefits?

Reply:
Ok fixed, was missing `objectMode: true`. Transforms are not really well documented on Nodejs docs, cannot find a reference on transform options :confused:

Reply:
Should I remove through2 as dep and use Transform instead in all project? Has it better performance?